src/library/Grid/__tests__/Grid.spec.js

```diff
@@ -0,0 +1,24 @@
+/* @flow */
+import React from 'react';
+import { shallow } from 'enzyme';
+import testDemoExamples from '../../../../utils/testDemoExamples';
+import Grid, { GridItem } from '../../Grid';
+import examples from '../../../website/app/demos/Grid/examples/Grid';
+
+function shallowGrid([gridProps, gridItemProps]) {
+  return shallow(
+    <Grid {...gridProps}>
+      <GridItem {...gridItemProps} />
+    </Grid>
+  );
+}
+
+describe('Grid', () => {
+  it('renders', () => {
+    const grid = shallowGrid([{}, {}]);
```

Comment:
While it really doesn't matter, there is no reason to pass an array here.  You could simply pass 2 props objects.